.gitlab-ci.yml

```diff
@@ -126,7 +126,7 @@ check_tag:
   script:
     - git config --global --add safe.directory "${CI_PROJECT_DIR}"
     - git fetch --tags
-    - tag=release_$(grep -w __version__ scargo/scargo_src/global_values.py | grep -Eo "[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+")
+    - tag=release_$(grep -w __version__ scargo/__init__.py | grep -Eo "[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+")
```

Comment:
please do not change it, it deliver version to setup.py but also to or implementation so we maintain only one place

Reply:
It will still be in one place, just done differently. It will only be in __init__.py
https://flit.pypa.io/en/stable/pyproject_toml.html